AdyenDropIn/Views/PreApplePayView.swift

```diff
@@ -26,15 +26,16 @@ internal final class PreApplePayView: UIView, Localizable {
     
     /// Creates PKPaymentButtonStyle based on Dark or Light Mode.
     private var paymentButtonStyleAuto: PKPaymentButtonStyle {
-        let buttonStyle: PKPaymentButtonStyle
         if #available(iOS 14.0, *) {
-            buttonStyle = .automatic
-        } else if #available(iOS 12.0, *), traitCollection.userInterfaceStyle == .dark {
-            buttonStyle = .white
-        } else {
-            buttonStyle = .black
+            return .automatic
+        }
```

Comment:
we still have iOS13 right ?
What about `traitCollection.userInterfaceStyle == .dark` ?

Reply:
That's handled by the lines below 🤔